content/blog/2018/10/2018-10-12-gsoc2018-results.adoc

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: "Jenkins in Google Summer of Code 2018. Results"
+tags:
+- community
+- events
+- gsoc
+- gsoc2018
+author: oleg_nenashev
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+It has been a while since the last blogpost about link:https://summerofcode.withgoogle.com/[Google Summer of Code] in Jenkins.
+GSoC 2018 has officially finished on August 23, and we had a link:https://www.meetup.com/Jenkins-online-meetup/events/253577758/[Jenkins Online Meetup] where we had final presentations of the GSoC projects.
+It is never late to provide more context, so I would like to summarize the results and provide updates of what was happening in link:/sigs/gsoc[Jenkins GSoC Special Interest Group] over last 2 months. 
+In this blogpost you can find project status overviews and updates from the Jenkins link:/sigs/gsoc[GSoC SIG].
+
+But first of all, I would like to thank all our students, their mentors
+and to all other contributors who proposed project ideas, participated in student selection, community bonding and further reviews.
+Google Summer of Code is a major effort which would not be possible without it.
```

Comment:
"without it" seems too vague, how about: not possible without the active participation of the Jenkins community.